recipe-client-addon/lib/RecipeRunner.jsm

```diff
@@ -174,8 +174,6 @@ this.RecipeRunner = {
         resolve(clonedResult);
       });
       sandboxManager.addGlobal("actionFailed", err => {
-        Cu.reportError(err);
```

Comment:
I expect you could still report the fake-cloned error you create a few lines lower down? Which might still be helpful, but eh, I don't really mind too much. Up to you whether you want to bother with that or not.